backend/src/LibBackend/Account.fs

```diff
@@ -22,24 +22,9 @@ type Account =
     email : string
     name : string }
 
-type UserInfo =
-  { username : UserName.T
-    name : string
-    admin : bool
-    email : string
-    id : UserID }
-
-type UserInfoAndCreatedAt =
-  { username : UserName.T
-    name : string
-    admin : bool
-    email : string
-    id : UserID
-    createdAt : NodaTime.Instant }
-
-let userInfoToPerson (ui : UserInfo) : LibService.Rollbar.Person =
-  Some { id = ui.id; username = Some ui.username }
+type UserInfo = { id : UserID }
```

Comment:
I think we can remove this type entirely.

Reply:
Removing it goes way deeper
It used by `testOwner` in `TestUtils.fs` which is used by `initializeTestCanvas` and `createTestCanvas` which are used by 

- Canvas.Tests
- Fuzz tests ( Passowirds.FuzzTests, ExecutionRegression.fuzzTests.fs, ExecutePureFunctions.FuzzTests.fs)
- Cron.Tests
- Stdlib.Tests,
- Traces.Tests
- `execute` function in FSI.fs, 
etc.

Reply:
I have tried replacing
```
`let getUser (username : UserName.T) : Task<Option<UserInfo>>
// by 
let getUser (username : UserName.T) : Task<Option<UserID>>  

// and 
(owner : Task<LibBackend.Account.UserInfo>)
// by 
(owner : Task<UserID>)

// and 
let initializeCanvasForOwner
 (owner : Account.UserInfo)
 (name : string)
// by
let initializeCanvasForOwner
 (owner : UserID)
 (name : string)
```

but the backend tests were not passing
example of the errors :
```
account_id = 77a... is not present in table accounts

Unexpected exception count in test test-htdjy
```

Reply:
Seems like you might need to edit some of the SQL to reflect these changes.